test/extended/images/mongodb_ephemeral.go

```diff
@@ -0,0 +1,64 @@
+package images
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/db"
+	"time"
+)
+
+var _ = g.Describe("images: mongodb: ephemeral template", func() {
+	defer g.GinkgoRecover()
+	var (
+		templatePath = exutil.FixturePath("..", "..", "examples", "db-templates", "mongodb-ephemeral-template.json")
+		oc           = exutil.NewCLI("mongodb-create", exutil.KubeConfigPath()).Verbose()
```

Comment:
Why not use `:=`? It would make it  shorter and less indented:

```go
templatePath := exutil.FixturePath("..", "..", "examples", "db-templates", "mongodb-ephemeral-template.json")
oc := exutil.NewCLI("mongodb-create", exutil.KubeConfigPath()).Verbose()
```